lib/airbrake/sidekiq/retryable_jobs_filter.rb

```diff
@@ -13,6 +13,10 @@ class RetryableJobsFilter
         DEFAULT_MAX_RETRY_ATTEMPTS = ::Sidekiq::JobRetry::DEFAULT_MAX_RETRY_ATTEMPTS
       end
 
+      def initialize(notify_after: nil)
```

Comment:
It's hard to guess that this option expects a number. It looks more like a callback to me, so I would think it wants a Proc. The option is also related to retries but nothing in the name suggests that. I believe we need to find a better fit.

Maybe it could be: `max_retries`? We would set it to the `max_retries` method call by default. Then the `max_attempts_for` would use `@max_retries` instead of `max_retries`. Does that make sense?

P.S. We need unit tests here as well.